hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -27,6 +28,7 @@ const Orders = () => {
     };
 
     useEffect(() => {
+        dispatch(clearFilters());
```

Comment:
hmm I'm not so sure we want to clear filters every time we load the page. It might be nice for the user to go back to exactly the same orders they were looking at before.